server/models/captureService.js

```diff
@@ -0,0 +1,95 @@
+const { knex: db } = require('../database/knex');
+const { v4: uuidv4 } = require('uuid');
+
+// DATABASE QUERIES
+
+async function getCaptures(req, res) {
```

Comment:
While the functions here are straight forward and works , it would be nice to have infrastructure, model layer separation. If you look at field-data service, we have the repositories that does the querying and inserting data to the tables. The model layer does the business logic and service/routes orchestrates the model to meet a use case.

I can pair program with you to show what I mean.